controllers/contentlibrary/clustercontentlibraryitem/clustercontentlibraryitem_controller.go

```diff
@@ -202,6 +202,8 @@ func (r *Reconciler) setUpCVMIFromCCLItem(cvmi *vmopv1a1.ClusterVirtualMachineIm
 		return err
 	}
 
+	cvmi.Labels = cclItem.Labels
```

Comment:
We don't want to just replace `labels` wholesale, right? What if there are labels on the images not on the CL items?

Reply:
makes sense. We only wanna watch for service type labels from cl items.